data/yam/agama/auto/lib/scripts_post.libsonnet

```diff
@@ -6,5 +6,13 @@
       #!/usr/bin/env bash
       echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
     |||
+  },
+  enable_gdm: {
```

Comment:
this step should be drop once the fix is in the build, have you checked with latest build if still needed the workaround?

Reply:
tested in prod and still [needed](https://openqa.suse.de/tests/17454237).